react-js/styling-react.md

```diff
@@ -0,0 +1,60 @@
+# Styling in ReactJS
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Independent Practice: 20 mins.
+
+### Prerequisites
+- CSS: [css](../css/css.md)
```

Comment:
Can you title these the way their respective files are titled, and add the word "Outline" or "Lesson"?  Also, there's not need for `.` or `..`. 
example: 
-  [CSS Lesson](/css/css.md)